components/script/dom/Test_mapping.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can we move this file so it's not inside components?

Reply:
The idea is, every directory will have its own test_mapping.json which will have list of files in the directory with its test mapping.

The current logic to run mutation test is like this:
given some/path
read some/path/test_mapping.json
parse if mapping found
run mutation test for all files listed in this json.
for each some/path/otherFolders:
   repeat the above step recursively

Are you suggesting to have test mapping of all the source files in one test_mapping.json?